awx/ui_next/src/screens/Project/ProjectList/ProjectListItem.jsx

```diff
@@ -11,16 +14,24 @@ import {
 import { t } from '@lingui/macro';
 import { Link } from 'react-router-dom';
 import { PencilAltIcon, SyncIcon } from '@patternfly/react-icons';
+import styled from 'styled-components';
 
 import ClipboardCopyButton from '@components/ClipboardCopyButton';
-import DataListCell from '@components/DataListCell';
-import DataListCheck from '@components/DataListCheck';
 import ProjectSyncButton from '../shared/ProjectSyncButton';
 import { StatusIcon } from '@components/Sparkline';
-import VerticalSeparator from '@components/VerticalSeparator';
 import { toTitleCase } from '@util/strings';
 import { Project } from '@types';
 
+const PaddedIcon = styled(StatusIcon)`
```

Comment:
I feel like StatusIcon should work generally, pulling from @components/StatusIcon and it work in list without tweaks.  And we can import that into Sparkline and make tweaks there.